composer.json

```diff
@@ -19,7 +19,7 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
+        "php": "^7.1 || ~8.0.0 || ~8.1.0 || ~8.2.0",
```

Comment:
Careful with `8.2.0`: I suggest holding off until it reaches RC, around october 2022.

Until then, any BC break is still possible.